show/main.py

```diff
@@ -960,8 +960,16 @@ def version(verbose):
     asic_type = version_info['asic_type']
     asic_count = multi_asic.get_num_asics()
 
-    serial_number_cmd = "sudo decode-syseeprom -s"
-    serial_number = subprocess.Popen(serial_number_cmd, shell=True, text=True, stdout=subprocess.PIPE)
+    serial_number=''
```

Comment:
Add spaces around `=`. Also suggest assigning to `None` by default rather than empty string.

Reply:
> Add spaces around `=`. Also suggest assigning to `None` by default rather than empty string.

Done